pkgs/development/compilers/sbcl/default.nix

```diff
@@ -78,48 +85,63 @@ stdenv.mkDerivation rec {
     inherit (versionMap.${version}) sha256;
   };
 
-  nativeBuildInputs = [ texinfo ];
+  nativeBuildInputs = [
+    texinfo
+  ] ++ lib.optionals self.doCheck (
+    [
+      which
+    ] ++ lib.optionals (builtins.elem stdenv.system strace.meta.platforms) [
```

Comment:
`stdenv.system` or `stdenv.hostPlatform.system`?

Reply:
Hmmm, do we even test in cross-compilation anyway?

Reply:
Correction: are we ever going to have working SBCL cross-compilation anyway, given that the upstream level of support is «well you need to run _some_ parts of the process on the host arch and not on the build arch anyway». (My bet is not until a large — and not too likely — change upstream)